types/primitive.yaml

```diff
@@ -18,10 +18,9 @@ Version:
   example: "Lighthouse/v0.1.5 (Linux x86_64)"
 
 GenesisTime:
-  allOf:
-    - $ref: "./primitive.yaml#/Uint64"
-    - example: "1590832934"
-    - description: "The genesis_time configured for the beacon node, which is the unix time in seconds at which the Eth2.0 chain began."
+  $ref: "./primitive.yaml#/Uint64"
+  description: "The genesis_time configured for the beacon node, which is the unix time in seconds at which the Eth2.0 chain began."
+  example: "1590832934"
```

Comment:
Should make sure to use same order of properties everywhere, or how did you determine if description vs example should be 2nd property? Looking at `ExecutionOptimistic` below, the original order was swapped

Reply:
There is no consistent ordering in general. But that's probably for a different PR.
That said, I would be in favor of the following:
* `type` or `$ref`
* `description`
* `example`